CRM/Upgrade/Incremental/php/FiveSeventyNine.php

```diff
@@ -29,6 +29,28 @@ class CRM_Upgrade_Incremental_php_FiveSeventyNine extends CRM_Upgrade_Incrementa
    */
   public function upgrade_5_79_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Update "Website Type" options', 'updateWebsiteType');
+  }
+
+  /**
+   * Delete branded website-type options that are not in use.
+   * Add new "Social" option.
+   */
+  public static function updateWebsiteType() {
+    $query = CRM_Core_DAO::executeQuery("SELECT `id`, `value` FROM `civicrm_option_value`
+      WHERE `option_group_id` = (SELECT `id` FROM `civicrm_option_group` WHERE `name` = 'website_type')
```

Comment:
EXPLAIN says this would be better as a join but I don't think it will matter too much.

The `NOT IN (SELECT `website_type_id` FROM `civicrm_website`)` I thought might benefit from a group by website_type_id, but `explain` doesn't seem to care.

Also this makes me realize the sample data still has the old values. I'll put up a regen in a minute.

Reply:
Thanks @demeritcowboy